kedro/templates/project/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/settings.py

```diff
@@ -3,29 +3,27 @@
 https://kedro.readthedocs.io/en/stable/kedro_project_setup/settings.html."""
 
 # Instantiated project hooks.
-# from {{cookiecutter.python_package}}.hooks import ProjectHooks
-# HOOKS = (ProjectHooks(),)
+# HOOKS = (,)
```

Comment:
I don't know what's the best example here, since we don't have a built-in hook available. The strongest candidate is SparkHook but we only put it in the `pyspark-iris`.

The important think here is that you need an instance, not a class inside `HOOKS`

Reply:
@astrojuanlu

Reply:
What about this?

```
# For example, after creating a hooks.py and defining a ProjectHooks class there, do
# from {{cookiecutter.python_package}}.hooks import ProjectHooks
# HOOKS = (ProjectHooks(),)
```

so we keep the recommendation, but we add a bit more context to it.